testing/functional/tests/cypress/page-objects/student-objects/ApplicationObject.ts

```diff
@@ -193,15 +194,15 @@ export default class ApplicationObject {
   }
 
   myStudyPeriodIsNotListedCheckbox() {
-    return cy.get("#eoslne6-offeringnotListed");
+    return cy.get("#etvv9qs-programnotListed");
```

Comment:
Using the IDs for the form.io definitions can be tricky because they can be auto regenerated on not even be in sync for different program years, for instance. We should rely on items that are not likely to change like the _name_ that is also used in the backend and is not supposed to be changed or will be changed under controlled circumstances.

Reply:
Changes are done.

Reply:
Same from before.